build.spec

```diff
@@ -50,9 +51,10 @@ hiddenimports: list[str] = [
 ]
 
 if sys.platform == "linux":
-    # Needed files for better system tray support on Linux via pystray (AppIndicator backend).
```

Comment:
Please retain this comment

Also, can you please put these AppIndicator-related changes into a separate commit explaining the reason for the move to Ayatana's AppIndicator (i.e. the old one is deprecated), and finally `git rebase` your branch, melding some commits so the list of commits look cleaner? Thanks.

Reply:
Just to be sure i understand correctly, you want me to do a separate pr about the move to Ayatana's Appindicator?

Reply:
It's not necessary to make a new pull request, you could use `git rebase` to fix the list of commits, and then force-push later. However, if you've never done that before, I think it's safer leave it like this.

Reply:
nonon, i know how to rebase, i'm thinking about the "can you please put these AppIndicator-related changes into a separate commit explaining the reason for the move to Ayatana's AppIndicator" part

Reply:
That would be part of the rebase, too. Your commits would [look like this](https://github.com/DevilXD/TwitchDropsMiner/compare/master...guihkx:TwitchDropsMiner:example), for example.

Reply:
Sorry i misunderstood what you meant, should be all done now